vars/mavenCanaryRelease.groovy

```diff
@@ -22,7 +22,11 @@ def call(body) {
     sh "git checkout -b ${env.JOB_NAME}-${config.version}"
 
     if (autoUpdateFMP) {
-        patchFMPVersion()
+        try {
+            patchFMPVersion()
+        } catch (err) {
+            println "FMP patching failed due to : "+ err.message
```

Comment:
shouldn't this be echo?

Reply:
it can be but `prinln` can abstract it way better, it works in basic groovy.